packages/drivers/odsp-driver/src/compactSnapshotParser.ts

```diff
@@ -146,6 +146,21 @@ function readTreeSection(node: NodeCore) {
 				continue;
 			}
 
+			// "name": <node name>
+			// "children": <blob id>
+			// groupId: <group name>
+			if (
+				length === 6 &&
+				treeNode.getMaybeString(2) === "children" &&
+				treeNode.getMaybeString(4) === "groupId"
```

Comment:
I'm a bit lost here. I'd expect to find a "pair" for any block added, i.e. equivalent block without groupId (i.e. length === 5 && treeNode.getMaybeString(2) === "children"), but it's not there. Am I missing something?

Reply:
with name, children and groupid, length would be 6 from 0 to 5. So, 4 would be groupId and 5 would be its value.

Reply:
The pair with length 4 is on the top of switch. Case: "4"